pkg/kubectl/stop.go

```diff
@@ -78,19 +78,26 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string) (string,
 	return fmt.Sprintf("%s stopped", name), nil
 }
 
-func (reaper *PodReaper) Stop(namespace, name string) (string, error) {
+func (reaper *PodReaper) Stop(namespace, name string, gracePeriodSec int) (string, error) {
```

Comment:
I'm trying to get rid of stop. Delete now performs stop-like behavior by default. Please add this to delete as well.